src/main/java/Duke.java

```diff
@@ -1,10 +1,176 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.BufferedWriter;
+import java.io.OutputStreamWriter;
+import java.io.FileOutputStream;
+import java.util.ArrayList;
+import java.util.NoSuchElementException;
+import java.util.StringTokenizer;
+import java.nio.charset.StandardCharsets;
 public class Duke {
-    public static void main(String[] args) {
+    public static void main(String[] args) throws Exception {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        BufferedWriter out = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(
+                java.io.FileDescriptor.out), StandardCharsets.UTF_8));
+        out.write("Hello from\n" + logo);
+        out.write("Hello! I'm Duke\n" + "What can I do for you?\n");
+        out.flush();
+        String curr = br.readLine();
+        StringTokenizer st = new StringTokenizer(curr);
+        //String cmd = st.nextToken();
+        ArrayList<Task> tasks = new ArrayList<>(100);
+        Task currTask = null;
+        //Command cmd = Command.valueOf(st.nextToken());
+        while (true) {
+            try {
+                Command cmd = Command.valueOf(st.nextToken());
```

Comment:
Since you are using the enum class in your code, your code can be kept shorter and more readable if the behaviour of executing each Command is shifted under the respective enum instances. One possible idea would be to use an abstract method execute() in Command, and implement the execute method in each instance of the enum class:

enum Command {
    BYE{
        @Override
        public void execute() { ... }
    } , 
    LIST{
        @Override
        public void execute() { ... }
    } , ......

    public abstract void execute();
}

then the execution of each command in the main method is simply:  Command.valueOf(st.nextToken()).execute();